src/node_data.js

```diff
@@ -127,6 +122,10 @@ const getData = function(node) {
  * @return {!NodeData} The NodeData for this Node.
  */
 const importNode = function(node) {
+  if (node['__incrementalDOMData']) {
```

Comment:
we should really pull `__incrementalDOMData` out into a constant, accessed in 4 places